teams/wg-compiler-performance.toml

```diff
@@ -23,9 +23,10 @@ alumni = [
 orgs = ["rust-lang"]
 
 [website]
-name = "Compiler performance working group"
+name = "Compiler performance working area"
```

Comment:
Here we change working group to working area.

Reply:
It was like that originally, I suggested to add some suffix (https://github.com/rust-lang/team/pull/1763#issuecomment-2778353451).

Reply:
In any case, what I meant with my 3 comments is that it seems it lacks some consistency. In some place is working area, in some others is working group and in others nothing.
I was wondering if that have a reason that I'm not understanding.